ui/app/AppLayouts/Wallet/views/TransactionSettings.qml

```diff
@@ -60,6 +60,8 @@ Rectangle {
     required property var fnGetEstimatedTime
     required property var fnRawToGas
     required property var fnGasToRaw
+    required property var toLocaleStr
+    required property var fromLocaleStr
```

Comment:
nitpick - to preserve the component style
```suggestion
    required property var fnToLocaleStr
    required property var fnFromLocaleStr
```

Reply:
@alexjba updated.